english/coffee/drink-tea/be-brave/brave.md

```diff
@@ -0,0 +1 @@
+You fully reallized your foolishness, the cofee made the burning worse. However the energy gained from cofee gave you a stamina boost to run 3x times faster
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.